### PR TITLE
docs: add operate batch modification page

### PR DESCRIPTION
**Pull request by johanwelgemoed**
## Description

Closes https://github.com/camunda/camunda-docs/issues/3233. Related to https://github.com/camunda/operate/issues/5872.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineerring DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Review by vitorwtavares — DISMISSED** [review]

**Comment by johanwelgemoed** [comment]
> Can we add a note or info on the existing page that if a user is looking for the batch modification it exists? They may not know or look in the sidebar. Can you add something to https://docs.camunda.io/docs/components/operate/userguide/process-instance-modification/ that links to the new batch page?

Good suggestion. I'll add it now

File: docs/components/operate/userguide/process-instance-batch-modification.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+id: process-instance-batch-modification
+title: Process instance batch modification
+description: "You may need to move multiple active process instances to allow execution to continue."
+---
+
+If there was an issue in process execution that caused you to enter the wrong process branch or data was corrupted, you may need to select multiple process instances and move them to the correct flownode in the process in a single operation.
+
+:::note
+If you only need to modify a single active process instance to allow execution to continue, instead use [Process instance modification](./process-instance-modification.md), which also provides some additional modification options beyond move modification.
```
Comment (johanwelgemoed) [inline docs/components/operate/userguide/process-instance-batch-modification.md:10]: @akeller would this be sufficient?

File: docs/components/operate/userguide/process-instance-modification.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ description: "You may need to modify an active process instance to allow executi
 
 You may need to modify an active process instance to allow execution to continue. The execution may be stuck, and you may want to continue the execution on a different activity (i.e. skip or repeat activities).
 
+:::note
+If an issue in process execution caused you to enter the wrong process branch, instead of moving each process instance individually, instead use [Process instance batch modification](./process-instance-batch-modification.md) to select multiple process instances and move them to the correct flownode in the process in a single operation.
```
Comment (johanwelgemoed) [inline docs/components/operate/userguide/process-instance-modification.md:10]: @akeller also added note here. would this be sufficient?

**Review by christinaausley — APPROVED** [review]

**Comment by johanwelgemoed** [comment]
@akeller I am not sure, but I think this still waits for additional review from you? OR is there anything else on my end I need to add?

**Review by akeller — APPROVED** [review]

**Comment by akeller** [comment]
Looks good! @christinaausley's blessing is far worthier than mine 😄 

If this is ready to merge, let's do it!

**Pull request merged**